all2vec/__init__.py

```diff
@@ -229,6 +229,16 @@ def get_similar_threshold(self, entity_type, entity_id, match_type,
             return self.get_similar_threshold(
                 entity_type, entity_id, match_type, threshold, n_try*10)
 
+    def get_entity_types(self):
+        """Helper for getting entity types object."""
+        return [{
+            'num_entities': etype._ann_obj.get_n_items(),
+            'entity_type_id': etype._entity_type_id,
+            'entity_type': etype._entity_type,
+            'metric':  etype._metric,
+            'num_trees': etype._ntrees
+        } for k, etype in self._annoy_objects.items()]
```

Comment:
Is `k` actually needed here? If not, you can use `self._annoy_objects.values()` instead.